lib/fizzy/parser.cpp

```diff
@@ -64,6 +64,12 @@ parser_result<ValType> parse(const uint8_t* pos, const uint8_t* end)
     }
 }
 
+const uint8_t* validate_valtype(const uint8_t* pos, const uint8_t* end)
```

Comment:
Shouldn't this be inline?

Reply:
It cannot, because is used in other compilation unit.

Reply:
Oh thought this was the header. Nvm.